lib/jasmine.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) 2018 Hetzner Cloud GmbH
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+declare namespace jasmine {
+    interface Matchers<T> {
+        toBePresent(): boolean;
```

Comment:
Please add documentation about the difference between being `present` and being `displayed`